File: vinhdinh/ARRAY.pas
uses crt;

var n, i, maxn, maxp, temp:integer;

begin
	readln(n);
	maxn := 0; maxp := 0;
	for i := 1 to n do
		begin
			read(temp);
			if temp >= maxn then
				begin
					maxn := temp;
					maxp := i;
				end;
		end;
	write(maxn); write(' '); write(maxp);
end.

File: SolVTTS/ARRAY.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){
	int n, maxn, maxp; 
	cin >> n;
	for (int i = 1; i <= n; i++){
		int temp;
		cin >> temp;

		if (temp >= maxn){
			maxn = temp;
			maxp = i;
		}
	}	

	cout << maxn << " " << maxp;
	return 0;
}